src/initializers/koa/riviere.ts

```diff
@@ -31,5 +31,9 @@ export default (config, orkaOptions: Partial<OrkaOptions>) =>
     bodyKeys: config.riviere.bodyKeys,
     bodyKeysRegex: config.riviere.bodyKeysRegex && new RegExp(config.riviere.bodyKeysRegex, 'i'),
     color: config.riviere.color,
-    context: orkaOptions.riviereContext
+    context: (ctx: Koa.Context) => { 
+      return {
+        visitor: ctx.state.visitor,
```

Comment:
❓ @klouvas Does riviere handle null, undefined values?

Reply:
yes. in both json & extended styles